core/src/main/java/org/apache/carbondata/core/util/SessionParams.java

```diff
@@ -153,6 +154,12 @@ private boolean validateKeyValue(String key, String value) throws InvalidConfigu
       case ENABLE_UNSAFE_IN_QUERY_EXECUTION:
       case ENABLE_AUTO_LOAD_MERGE:
       case CARBON_PUSH_ROW_FILTERS_FOR_VECTOR:
+      case CARBON_LOAD_SETLENIENT_ENABLE:
```

Comment:
It can be  a fall through case. Can remove line 158-162`

Reply:
done